app/controllers/admin/dashboard_controller.rb

```
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Admin
+  class DashboardController < ApplicationController
+    before_action :require_admin
+
+    def index
+    end
```

RuboCop (1.84.2)

[convention] Put empty method definitions on a single line.

(Style/EmptyMethod)